app/src/main/java/org/mozilla/focus/session/SessionManager.java

```diff
@@ -230,6 +230,15 @@ public void removeCurrentSession() {
         removeSession(currentSessionUUID);
     }
 
+    /**
+     * Remove the session by its UUID.
+     */
+    public void removeSessionByUUID(@NonNull int position) {
```

Comment:
This actually removes the session by index/position and not by UUID. That's the method below.

Reply:
Renamed it to removeSessionByPosition. 👍